microservices/nris_api/openshift/_python36_oracle.dc.json

```diff
@@ -0,0 +1,444 @@
+                      "secretKeyRef": {
+                        "name": "${DB_CONFIG_NAME}",
+                        "key": "database-nris-user"
+                      }
+                    }
+                  },
+                  {
+                    "name": "DB_HOST",
+                    "valueFrom": {
+                      "secretKeyRef": {
+                        "name": "${DB_CONFIG_NAME}",
+                        "key": "database-host"
+                      }
+                    }
+                  },
+                  {
+                    "name": "DB_PASS",
+                    "valueFrom": {
+                      "secretKeyRef": {
+                        "name": "${DB_CONFIG_NAME}",
+                        "key": "database-nris-password"
+                      }
+                    }
+                  },
+                  {
+                    "name": "DB_PORT",
+                    "value": "5432"
+                  },
+                  {
+                    "name": "DB_NAME",
+                    "valueFrom": {
+                      "secretKeyRef": {
+                        "name": "${DB_CONFIG_NAME}",
+                        "key": "database-name"
+                      }
+                    }
+                  },
+                  {
+                    "name": "NRIS_DB_HOSTNAME",
+                    "valueFrom": {
+                      "secretKeyRef": {
+                        "name": "${MDS_NRIS_ODB_CREDENTIALS}",
+                        "key": "db-host"
+                      }
+                    }
+                  },
+                  {
+                    "name": "NRIS_DB_SERVICENAME",
+                    "valueFrom": {
+                      "secretKeyRef": {
+                        "name": "${MDS_NRIS_ODB_CREDENTIALS}",
+                        "key": "db-service"
+                      }
+                    }
+                  },
+                  {
+                    "name": "NRIS_DB_USER",
+                    "valueFrom": {
+                      "secretKeyRef": {
+                        "name": "${MDS_NRIS_ODB_CREDENTIALS}",
+                        "key": "db-user"
+                      }
+                    }
+                  },
+                  {
+                    "name": "NRIS_DB_PASSWORD",
+                    "valueFrom": {
+                      "secretKeyRef": {
+                        "name": "${MDS_NRIS_ODB_CREDENTIALS}",
+                        "key": "db-password"
+                      }
+                    }
+                  },
+                  {
+                    "name": "NRIS_DB_PORT",
+                    "value": "1521"
+                  },
+                  {
+                    "name": "ELASTIC_ENABLED",
+                    "value": "${ELASTIC_ENABLED}"
+                  },
+                  {
+                    "name": "ELASTIC_SERVICE_NAME",
+                    "value": "${ELASTIC_SERVICE_NAME}"
```

Comment:
As for the question with APM above, im fairly certain the service name is what differentiates different application on APM.
@itchison may know better tho

Reply:
APM is going to be coming in later, i believe you are correct and i plan on tackling that next PR.